core/put/source/dev_ui.cpp

```diff
@@ -417,21 +417,21 @@ namespace put
             style.Colors[ImGuiCol_HeaderHovered] = foreground_dark_highlight;
             style.Colors[ImGuiCol_HeaderActive] = foreground;
 
-            style.Colors[ImGuiCol_Column] = foreground_dark;
-            style.Colors[ImGuiCol_ColumnHovered] = foreground_light;
-            style.Colors[ImGuiCol_ColumnActive] = foreground;
+            // style.Colors[ImGuiCol_Column] = foreground_dark;
```

Comment:
if the enums are now changed can we find out what the new names are and re-instate this commented out code?

Reply:
I will do so. I do think they removed it entirely.